### DISCUSSION TITLE
No Print Statements inside lib/sqlalchemy/testing/plugin/pytestplugin.py

### DISCUSSION THREAD

**Discussion by Kajalagg98** [sqlalchemy/sqlalchemy#7307] [Usage Questions] [answered]
print() statements doesn't seems to work inside sqlalchemy/lib/sqlalchemy/testing/plugin/pytestplugin.py. Any way to make them work ?

**Comment by zzzeek** [comment, marked as answer]
pytest suppresses stdout unless you pass -s to the pytest commadn

**Reply by Kajalagg98** [reply]
Got it. Thanks!
